Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>Re: our conv - look for workspace/sub details in msticpyconfig.yaml - esp since you've connected the qry_provider to a workspace above.</p><p>Maybe either get list of all configured workspaces (maybe ones that have a subscriptionId) or default to the default if there's only one.</p><p> </p><p>Maybe this is only showing for default resource group (if there is such a thing). If I manually build a resource ID with the right sub and resource and workspace it works ok.</p><p> </p><p>Another thing - not specific to the notebook.</p><p>I got an access token timeout while working on this. I wonder if there's anything we can do to auto-refresh this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>